tools/packaging/packaging.psm1

```diff
@@ -2212,6 +2248,24 @@ function New-MSIPatch
     Remove-Item -Path $filesToCleanup -Force -Recurse -ErrorAction SilentlyContinue
 }
 
+<#
+    .Synopsis
+        Tests if a version is preview
+    .EXAMPLE
+        Test-IsPreview -version '6.1.0-sometthing' # returns true
+        Test-IsPreview -version '6.1.0' # returns false
+#>
+function Test-IsPreview
```

Comment:
Only note here is that a string like `"6.0.0-alpha.13-1557-gd67d87c"` also passes this test. Just want to make sure that's intended behaviour (that string gets caught by the validation attribute so not a big concern).

Reply:
an alpha would be considered a preview.  basically anything with a dash (`-`) in the version is a preview.